runtime_scan/pkg/orchestrator/scanconfigwatcher/watcher.go

```diff
@@ -198,6 +199,7 @@ func (w *Watcher) createScan(ctx context.Context, scanConfig *models.ScanConfig)
 		Count:  utils.PointerTo(true),
 	})
 	if err != nil || scans == nil {
+		// TODO not found error is also an error here?
```

Comment:
@chrisgacsal The question is what if the error is NotFound? That is a legitimate error here and expected since we are looking to see if exists such scan, and if not we create it.

Reply:
There is no need to modify here, the code inside the provider should be handling any specific errors like not found and then either returning an error if its an error, a RetryableError if its to do requeued, or a FatalF if its fatal. GCP support should not touch *anything* outside of provider/gcp now

Reply:
@sambetts-cisco I did not change anything as you can see. This is only a comment. Not sure I understand how NotFound error is being handled here.

Reply:
This is a GET on endpoint api/scans, there should never be a "NotFound" unless the backend doesn't serve api/scans. If nothing matches the filter then you just get an response like `{ "Items": [] }`.